src/Razor/test/Microsoft.AspNetCore.Razor.LanguageServer.Test/DocumentSymbols/DocumentSymbolEndpointTest.cs

```diff
@@ -22,7 +22,7 @@ public class DocumentSymbolEndpointTest(ITestOutputHelper testOutput) : SingleSe
     public Task DocumentSymbols_CSharpMethods()
         => VerifyDocumentSymbolsAsync(
             """
-            @functions {
+            {|AspNetCore.test:|}@functions {
```

Comment:
Curious what this is for?

Reply:
That's just the component source mapping surfacing here - the class `AspNetCore.test` is mapped to this zero-length snippet at the beginning of the razor (the emptyness between `:|`).